src/app/locationInfo/LocationInfo.jsx

```diff
@@ -68,14 +55,26 @@ class LocationInfo extends Component {
           title="Location Info" />
         <ProgressBar step={0} steps={routes.length} />
         <LocationHeader />
-        {routes.map((route, i) => (
-          <FieldItem
-            key={route[0]}
-            title={route[2]}
-            linkTo={`${this.props.location.pathname}/${route[0]}`}
-            updatedAt={FAKE_DATA[i]}
-          />
-        ))}
+        {
+          routes.map((route, i) => {
+            const prop1 = route[3];
+            const prop2 = route[4];
```

Comment:
Feels a bit arbitrary that we assume all fields will always be nested exactly 2 levels deep. An array or dot-delimited string would probably make for a more general representation of paths, and, as you mentioned, allow us to just use a standard "get path" function, from lodash or underscore or whatever.

Reply:
Although, now that I think about it, the last property would always need to be a special case, since it's accessing an array of elements with field_names, as opposed to an object where the field_names are the props.
So, maybe a more generic solution wouldn't be that useful after all.

Perhaps just naming these something a bit more indicative would be nice though, like "metadata section" and "field name" instead of "prop 1" and "prop 2".

Reply:
Renamed prop1 and prop2 to metaDataSection and fieldName in b87dfec18c06b679b1b49a52590e15f67a00c5e5